lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,38 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
+      end
+
+      ALL = [
+        "Enable verbose logging with #{highlight('set verbose true')}",
```

Comment:
Though datastore options are functionally case-insensitive, I think they should be case-sensitive in documentation. There has been code in Framework that treats the options in a case-sensitive manner, too.